components/homme/src/share/bndry_mod_base.F90

```diff
@@ -969,97 +977,99 @@ subroutine sort_neighbor_buffer_mapping(par,elem,nets,nete)
   integer :: nets,nete
   type (ghostBuffer3D_t)   :: ghostbuf_cv
 
-  real (kind=real_kind) :: cin(2,2,4,nets:nete)                    ! 1x1 element input data
-  real (kind=real_kind) :: cout(2,2,4,max_neigh_edges+1,nets:nete)   ! 1x1 element output data
+  real (kind=real_kind) :: cin(2,2,4)                      ! 1x1 element input data
+  real (kind=real_kind) :: cout(2,2,4,max_neigh_edges+1)   ! 1x1 element output data
```

Comment:
at first glance, I was worried about this change because with the edge_pack routines, the element interior data is not packed/unpacked, and thus the output array often needs a nets:nete index to save the interior data.  

but here, it is only edge data - no interior data.  looks like this index was never needed.  good find!